### ISSUE TITLE
Cannot inherit from non-open class 'FloatingPanelController' outside of its defining module

### ISSUE THREAD

**Issue by rainypixels** [SCENEE/FloatingPanel#195]
### Short description
`FloatingPanelController` Subclassing still doesn't seem to be available. Xcode shows the error "Cannot inherit from non-open class 'FloatingPanelController' outside of its defining module". I can also confirm that the access level for the `FloatingPanelController` class definition still shows `public`. Perhaps carthage isn't pulling things down properly?

### Expected behavior
`FloatingPanelController` should be subclassable per #173. 

### Actual behavior
`FloatingPanelController` can't be subclassed.

### Steps to reproduce

```
class MyFloatingPanelController: FloatingPanelController {
 ...
}
```

### Environment

**Library version**
1.5.0

**Installation method**

- [ ] CocoaPods
- [x] Carthage
- [ ] Git submodules

**iOS version(s)**
Simulator / iOS 12.2

**Xcode version**
10.2

**Comment by SCENEE** [comment]
v1.5.0 doesn't have the fix for #173.  Today I've released v1.5.1! Please try the version 👍

**Comment by rainypixels** [comment]
Great, that worked! Thank you!
